client/components/oauth/oauth.tsx

```diff
@@ -25,26 +25,34 @@ interface Props {
 }
 
 const OAuth = ({ history }: Props) => {
-  const [cookie, setCookie] = useCookies(['jwt']);
+  const [cookie, setCookie, removeCookie] = useCookies(['jwt']);
 
   const dispatch = useDispatch();
 
   const oAuthSuccess = (response: GoogleLoginResponse) => {
     setCookie('jwt', response.getAuthResponse().id_token, { path: '/' });
-    const decodedJwt = jwtDecode(cookie.jwt);
-    dispatch(setUser(decodedJwt));
-    history.push('/success');
+    if (cookie.jwt) {
```

Comment:
I'm not following this.

Reply:
I don't think the cookie was necessarily set by the time the condition was hit, and it would result in an invalid token error. It shouldn't matter much though, because it makes more sense to use the Google response there anyway.